docs/send-data/opentelemetry-collector/install-collector-macos.md

```diff
@@ -48,7 +48,7 @@ You can run the script in two ways:
 
 * By piping `curl` straight into `bash`:
    ```bash
-   sudo curl -s https://github.com/SumoLogic/sumologic-otel-collector/releases/latest/download/install.sh | sudo -E bash -s -- --tag "host.group=default" --tag "deployment.environment=default" && sudo otelcol-sumo --config=/etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+   curl -Ls https://github.com/SumoLogic/sumologic-otel-collector/releases/latest/download/install.sh | sudo -E bash -s -- --tag "host.group=default" --tag "deployment.environment=default" && sudo otelcol-sumo --config=/etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
```

Comment:
When I run this command, it didn't wait for me to provide a password, and I received the following output:
```
% curl -Ls https://github.com/SumoLogic/sumologic-otel-collector/releases/latest/download/install.sh | sudo -E bash -s -- --tag "host.group=default" --tag "deployment.environment=default" && sudo otelcol-sumo --config=/etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
Password:
zsh: done                   curl -Ls  |
zsh: suspended (tty input)  sudo -E bash -s -- --tag "host.group=default" --tag
```

Reply:
Hm this is weird. This command "works on my machine" ™️  with zsh 5.9, but gives the above error on Dominika's with Mac's vanilla Zsh 5.8.1. 

I guess for now I will restore the `sudo`s in front of `curl`s in MacOS instructions, we can investigate this independently. For now, the most important thing is to add the missing curl redirection flag `-L` to the MacOS installation command.

Reply:
Hm this is weird. This command "works on my machine" ™️  with zsh 5.9, but gives the above error on Dominika's with Mac's vanilla Zsh 5.8.1. 

I guess for now I will restore the `sudo`s in front of `curl`s in MacOS instructions, we can investigate this independently. For now, the most important thing is to add the missing curl redirection flag `-L` to the MacOS installation command.

Reply:
We should test it on `bash` and not `zsh`

Reply:
Restored the `sudo`s.

Reply:
> We should test it on `bash` and not `zsh`

The MacOS instructions should probably be tested on the default MacOS's shell.

Reply:
yeeees, I mixed testing instructions vs testing installation script